packages/experimental/Avatar/src/JSAvatarTokens.win32.ts

```diff
@@ -0,0 +1,272 @@
+    },
+    inactive: {
+      avatarOpacity: 0.8,
+    },
+    size20: {
+      width: 20,
+      height: 20,
+      badgeSize: 'smallest',
+      iconSize: 16,
+      fontSize: globalTokens.font.size[100],
+      square: {
+        borderRadius: globalTokens.corner.radius.small,
+      },
+    },
+    size24: {
+      width: 24,
+      height: 24,
+      badgeSize: 'smallest',
+      iconSize: 16,
+      fontSize: globalTokens.font.size[100],
+      square: {
+        borderRadius: globalTokens.corner.radius.small,
+      },
+    },
+    size28: {
+      width: 28,
+      height: 28,
+      badgeSize: 'smaller',
+      iconSize: 20,
+      fontWeight: globalTokens.font.weight.semibold,
+      fontSize: globalTokens.font.size[100],
+    },
+    size32: {
+      width: 32,
+      height: 32,
+      badgeSize: 'smaller',
+      iconSize: 20,
+      fontSize: globalTokens.font.size[100],
+    },
+    size36: {
+      width: 36,
+      height: 36,
+      badgeSize: 'smaller',
+      iconSize: 20,
+    },
+    size40: {
+      width: 40,
+      height: 40,
+      badgeSize: 'small',
+      iconSize: 20,
+    },
+    size48: {
+      width: 48,
+      height: 48,
+      badgeSize: 'small',
+      iconSize: 24,
+    },
+    size56: {
+      width: 56,
+      height: 56,
+      badgeSize: 'medium',
+      iconSize: 28,
+      fontSize: globalTokens.font.size[400],
+      square: {
+        borderRadius: globalTokens.corner.radius.large,
+      },
+    },
+    size64: {
+      width: 64,
+      height: 64,
+      badgeSize: 'large',
+      iconSize: 32,
+      fontSize: globalTokens.font.size[500],
+      square: {
+        borderRadius: globalTokens.corner.radius.large,
+      },
+    },
+    size72: {
+      width: 72,
+      height: 72,
+      badgeSize: 'large',
+      iconSize: 32,
+      fontSize: globalTokens.font.size[500],
+      square: {
+        borderRadius: globalTokens.corner.radius.large,
+      },
+    },
+    size96: {
+      width: 96,
+      height: 96,
+      badgeSize: 'largest',
+      iconSize: 48,
+      fontWeight: globalTokens.font.weight.regular,
+      fontSize: globalTokens.font.size[700],
+      square: {
+        borderRadius: globalTokens.corner.radius.extraLarge,
+      },
+    },
+    size120: {
+      width: 120,
+      height: 120,
+      badgeSize: 'largest',
+      iconSize: 48,
+      fontWeight: globalTokens.font.weight.regular,
+      fontSize: globalTokens.font.size[900],
+      square: {
+        borderRadius: globalTokens.corner.radius.extraLarge,
+      },
+    },
+    neutral: {
+      backgroundColor: t.colors.neutralBackground6,
+      color: t.colors.neutralForeground3,
+      ringColor: t.colors.transparentStroke,
+    },
+    brand: {
+      backgroundColor: t.colors.brandBackgroundStatic,
+      color: t.colors.neutralForegroundOnBrand,
+      ringColor: t.colors.transparentStroke,
+    },
+    darkRed: {
+      ...getColorProps('darkRed', t),
+    },
```

Comment:
nit: I feel like you could do `darkRed:  getColorProps('darkRed', t),` for the code here instead of doing the spread operation on the return value of getColorProps. Seems a little cleaner to me.

Reply:
Thanks for catching this :)